src/index.html

```diff
@@ -48,5 +48,29 @@ <h2 class="mb-2">Question</h2>
         </ul>
       </section>
     </main>
+
+    <footer class="footer">
+      <div class="footer__container py-2">
```

Comment:
Is this `div` necessary or could you give the styles to the `footer`?

Reply:
Also, maybe look into accessibility and aria-label so that your navigation is accessible and readable. Right now I know this is a footer but I don't know, what is happening. Is it a navigation? :)